packages/fast-components-react-base/src/hypertext/hypertext.props.ts

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+
+/**
+ * Hypertext HTML tags
+ */
+export enum HypertextHTMLTags {
+    a = "a",
+}
+
+export interface IHypertextProps {
+
+    id?: string;
```

Comment:
I'm not sure this needs to be a first class prop for this component. I think we can pull this